packages/cozy-mespapiers-lib/src/assets/icons/Konnectors.svg

```diff
@@ -0,0 +1,21 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
In my opinion, the PNG assets should not be included inside repository. That reflexion can go inside a dedicated issue, I guess

Reply:
Mardi ok